client/src/app/site/agenda/models/view-item.ts

```diff
@@ -137,4 +138,17 @@ export class ViewItem extends BaseViewModel {
         }
         return false;
     }
+
+    public getSlide(): ProjectorElementBuildDeskriptor {
```

Comment:
If you want to project the content object, maybe a simple `return this.contentObject.getSlide()` is enough. Two things: Make shure, that contentObject is not null and that `isProjectable(this.contentObject)` is true.

Reply:
Whats about the case, if one of the assertions above fails? If you return null, make sure to catch this in the projector button, etc.